qiskit/visualization/circuit/_utils.py

```diff
@@ -499,42 +499,72 @@ def _sorted_nodes(dag_layer):
     return nodes
 
 
-def _get_gate_span(qubits, node, measure_arrows):
+def _get_gate_span(qubits, clbits, node, measure_arrows):
     """Get the list of qubits drawing this gate would cover
     qiskit-terra #2802
     """
-    min_index = len(qubits)
-    max_index = 0
+    wire_indices = []
     for qreg in node.qargs:
-        index = qubits.index(qreg)
+        try:
+            wire_indices.append(qubits.index(qreg))
+        except ValueError:
+            continue
+
+    offset = len(qubits)
+    for creg in node.cargs:
+        try:
+            wire_indices.append(offset + clbits.index(creg))
+        except ValueError:
+            continue
 
-        if index < min_index:
-            min_index = index
-        if index > max_index:
-            max_index = index
+    condition = getattr(node.op, "condition", None)
+    if condition is not None:
+        for bit in condition_resources(condition).clbits:
+            try:
+                wire_indices.append(offset + clbits.index(bit))
+            except ValueError:
+                continue
```

Comment:
We no longer support `Instruction(...).condition` in Qiskit. There are probably places in the code that still mention it, but it's no longer a thing.